lib/logstash/plugin_mixins/jdbc.rb

```diff
@@ -115,14 +115,16 @@ def jdbc_connect
           @logger.error("Failed to connect to database. #{@jdbc_pool_timeout} second timeout exceeded. Tried #{@connection_retry_attempts} times.")
           raise e
         else
-          @logger.error("Failed to connect to database. #{@jdbc_pool_timeout} second timeout exceeded. Trying again.")  
+          @logger.error("Failed to connect to database. #{@jdbc_pool_timeout} second timeout exceeded. Trying again.")
+          @metric_errors.increment(:connection_retries)
         end
       rescue Sequel::Error => e
         if retry_attempts <= 0
           @logger.error("Unable to connect to database. Tried #{@connection_retry_attempts} times", :error_message => e.message, )
           raise e
         else
           @logger.error("Unable to connect to database. Trying again", :error_message => e.message)
+          @metric_errors.increment(:connection_retries)
```

Comment:
The `retries` name here feels like not the right one. Maybe we call this just `:connection` ? So it may read as "connection errors" instead of "connection retry errors".